code/game/machinery/doors/windowdoor.dm

```diff
@@ -163,7 +163,7 @@
 
 	//If it's in the process of opening/closing, ignore the click
 	if (operating)
-		return
+		return TRUE
```

Comment:
this one's return TRUE, but firedoors were return only; should probably decide which behavior we want